cmd/erasure-server-pool.go

```diff
@@ -2322,6 +2322,10 @@ func (z *erasureServerPools) Health(ctx context.Context, opts HealthOptions) Hea
 	storageInfo := z.StorageInfo(ctx, false)
 
 	for _, disk := range storageInfo.Disks {
+		if disk.Local && opts.Maintenance {
```

Comment:
I am not sure why we are ignoring local disks here ? can`t a failed disk in the local node cause a wrong maintenance information ? Also not sure what bug will this fix

Reply:
Maintenance means I want to remove this node and do I have enough quorum to achieve that.

Read what we did earlier before the new code was implemented.

Reply:
oh right, I understand now